compiler/routes.py

```diff
@@ -0,0 +1,128 @@
+"""
+Provides the main API blueprint for compilation.
+
+Notes
+-----
+mypy doesn't have types for flask.Headers.extend, so those lines are
+excluded from type checking.
+
+"""
+
+from typing import Callable, Union, Iterable, Tuple, Optional
```

Comment:
There are a number of unused imports in this file.

Reply:
Removed in c1f323d